client/src/app/videos/+video-watch/video-watch-playlist.component.ts

```diff
@@ -119,6 +140,54 @@ export class VideoWatchPlaylistComponent {
       const start = next.startTimestamp
       const stop = next.stopTimestamp
       this.router.navigate([],{ queryParams: { videoId: next.video.uuid, start, stop } })
+    } else if (this.loopPlaylist) {
+      this.currentPlaylistPosition = 0
+      this.navigateToNextPlaylistVideo(this.playlistElements.find(e => e.position === this.currentPlaylistPosition))
```

Comment:
Please create a dedicated method to find the next element of a playlist depending on the position